contrib/IECoreMantra/src/IECoreMantra/procedural/Procedural.cpp

```diff
@@ -65,32 +65,6 @@ static void initialisePython()
 	}
 	Py_Initialize();
 	PyEval_InitThreads();
-		try
```

Comment:
I have to admit not understanding what's going on here very well, but I think we need to keep this to avoid re-doing the work in each `render()` call.